source/includes/fundamentals/code-snippets/authentication/aws-connection-string-session-token.go

```diff
@@ -8,5 +8,6 @@ assumeRoleCredential := options.Credential{
 		"AWS_SESSION_TOKEN": "<sessionToken>",
 	},
 }
-assumeRoleClient, err := mongo.Connect(context.TODO(),
-	options.Client().SetAuth(assumeRoleCredential))
\ No newline at end of file
+assumeRoleClient, err := mongo.Connect(
+	options.Client().SetAuth(assumeRoleCredential)
+)
```

Comment:
```suggestion
assumeRoleClient, err := mongo.Connect(options.Client().SetAuth(assumeRoleCredential))
```

Suggest making this a single line